src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,35 +44,41 @@ public String todayDate(DateTimePart datePart) {
      * Return Optional of a date built from these elements.
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
+        if (dateParams.length != 0
+                && dateParams[1] <= 12
+                && dateParams[2] <= 31) {
```

Comment:
what would be of day or mounth are less or equals 0? you should catch DateTimeException

Reply:
done